terraform/modules/jenkins/loadbalancer.tf

```diff
@@ -19,19 +19,20 @@ resource "aws_alb_target_group_attachment" "jenkins_target_attachment" {
   target_id        = aws_instance.jenkins.id
 }
 
-resource "aws_alb_target_group_attachment" "jenkins_api_target_attachment" {
-  target_group_arn = aws_alb_target_group.jenkins_api.arn
-  target_id        = aws_instance.jenkins.id
-}
-
 resource "aws_alb_target_group" "jenkins" {
   name     = "jenkins-target-group-${random_string.alb_prefix.result}-${var.environment}"
   port     = 80
-  protocol = "TCP"
+  protocol = "HTTP"
   vpc_id   = aws_vpc.main.id
-  stickiness {
-    enabled = false
-    type    = "lb_cookie"
+
+  health_check {
+    healthy_threshold   = "3"
+    interval            = "30"
+    protocol            = "HTTP"
+    matcher             = "301,200,403"
```

Comment:
Does the /login page return a 200? If so, could we use it as the healthcheck and remove the 403 response?

Reply:
Any ideas about this one?

Reply:
It does, I've changed the healthcheck.